src/App.jsx

```diff
@@ -14,33 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_CHAR = 'SORT_BY_CHAR';
+const SORT_BY_LENGTH = 'SORT_BY_LENGTH';
```

Comment:
move ur CONSTANTS to a separate directory and file - `utils/constants.ts`

Reply:
i think it would be better to create directory constants and their keep all constants,not in utils directory

Reply:
yep u're right, u can create another directory for constants. Good point 🔥